src/api/http-common.ts

```diff
@@ -5,6 +5,10 @@ export async function getEnvFromServer() {
   return await axios.get("/api/env").then((res: any) => res);
 }
 
+export async function getFilters() {
+  return await axios.get("/api/filters").then((res: any) => res);
+}
+
```

Comment:
All looks good, one thing - I am not sure what this endpoint does ?

Reply:
It is defined in the `pages` directory, under `/src/pages/api/filters.ts`. In Next you can build API routes: https://nextjs.org/docs/pages/building-your-application/routing/api-routes